server/pkg/asset/preview_type.go

```diff
@@ -13,6 +13,7 @@ const (
 	PreviewTypeGEO     PreviewType = "GEO"
 	PreviewTypeGEO3D   PreviewType = "GEO3D"
 	PreviewTypeMODEL3D PreviewType = "MODEL3D"
```

Comment:
Please lowercase all existing constant strings. The use of uppercase is for GraphQL convenience and is not common for constants used by Re:Earth. There is no impact because the conversion logic has already been properly implemented.

Reply:
Done!